lib/kie/service.rb

```diff
@@ -1,11 +1,17 @@
 require 'kie_client'
 module Kie
+  class NetworkError  < StandardError; end
```

Comment:
Move into `lib/exceptions.rb`. The error can be used by all external calls resulting network error or timeout.